model/Core/Vocabularies/RelationshipType.md

```diff
@@ -72,6 +72,7 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - republishedBy: (Security) Designates the agent that tracked, aggregated, and/or enriched vulnerability details to improve context (i.e. NVD)
 - requirementFor: Every `to` Element is required for the `from` Element
 - runtimeDependency: Every `to` Element is a runtime dependency for the `from` Element
+- spdxDocumentForArtfact: Every `to` Element is an SpdxDocument representing the logical serialization metadata for the `from`  Artifact containing the serliazed data
```

Comment:
```suggestion
- serializedArtifactFor: Every `to` Element is an artifact serialized from the `from` SpdxDocument Element
```

@sbarnum - when I tried to implement the suggested direction in the serialization example, it didn't seem to work - you would want to relate one logical SPDX document to multiple artifacts - not the other way around.  Thinking of changing this back.  What do you think?

Reply:
Oops, you are correct. I just realized that the directionality in the description is reversed. I misread it.

It should be 'from' and SpdxDocument 'to' an Artifact. 
This is what my initial comment above proposed the change be ("I would strongly recommend this RelationshipType be 'from' an "XCollection" 'to' an Artifact and not the other way around.").

I apologize for the confusion.